### PR TITLE
Add french Version

### PR DESCRIPTION
**Pull request by phrobinet**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe: 
Here is my French version of the documentation.

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch.
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing: https://github.com/vuejs/vue-test-utils/blob/dev/.github/CONTRIBUTING.md#development-setup
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**
```
Hello, 

Here is my French version of the documentation.
Do not hesitate to make a return if you want modifications.

Have a nice day
Philippe

```

### CODE REVIEW

**Comment by lmiller1990** [comment]
Wow, amazing! I will ping someone who knows French to give this a quick 👀

File: docs/fr/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Introduction
+
+Vue Test Utils est la bibliothèque officielle de tests unitaires pour Vue.js.
+
+Voici la documentation de Vue Test Utils v1, qui vise Vue 2 et les versions antérieures.
+
+En bref :
+
+- [Vue Test Utils 1](https://github.com/vuejs/vue-test-utils/) objectifs [Vue 2](https://github.com/vuejs/vue/).
```
Comment (Akryum) [inline docs/fr/README.md]:
```suggestion
- [Vue Test Utils 1](https://github.com/vuejs/vue-test-utils/) ciblant [Vue 2](https://github.com/vuejs/vue/).
```

File: docs/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- [Vue Test Utils 2](https://github.com/vuejs/vue-test-utils-next/) objectifs [Vue 3](https://github.com/vuejs/vue-next/).
```
Comment (Akryum) [inline docs/fr/README.md]:
```suggestion
- [Vue Test Utils 2](https://github.com/vuejs/vue-test-utils-next/) ciblant [Vue 3](https://github.com/vuejs/vue-next/).
```

File: docs/fr/api/components/README--.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Components
+
+Vue Test Utils comprend des composants utilitaires que vous pouvez utiliser pour avoir des composants stubs
```
Comment (Akryum) [inline docs/fr/api/components/README--.md]:
```suggestion
Vue Test Utils comprend des composants utilitaires que vous pouvez utiliser pour avoir des composants de remplacement (stubs).
```

File: docs/fr/api/config.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+Vue Test Utils comprend un objet de configuration des options définies utilisées par Vue Test Utils.
+
+### Vue Test Utils Config Options
+
+### `showDeprecationWarnings`
+
+- type: `Boolean`
+- default: `true`
+
+Contrôler s'il faut ou non afficher des avertissements de dépréciation . Lorsqu'il est sur `true`, tous les avertissements de déprédation sont visibles dans la console.
```
Comment (cexbrayat) [inline docs/fr/api/config.md]: déprédation -> déprécation

File: docs/fr/api/config.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+config.stubs['my-component'] = '<div />'
+```
+
+### `mocks`
+
+- type: `Object`
+- default: `{}`
+
+Comme pour les `stubs`, les valeurs passées à `config.mocks` sont utilisées par défaut. Toute valeur passées à l'objet d'options de montage `mocks` aura la priorité sur celles déclarées dans `config.mocks`.
```
Comment (cexbrayat) [inline docs/fr/api/config.md]: passées -> passée

File: docs/fr/api/createLocalVue.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  - `{Object} options`
+    - `{Function} errorHandler`
+
+- **Retours:**
+
+  - `{Component}`
+
+- **Usage:**
+
+`createLocalVue` renvoie une classe Vue pour que vous puissiez ajouter des composants, des mixins et installer des plugins dans polluer la classe Vue globale.
```
Comment (cexbrayat) [inline docs/fr/api/createLocalVue.md]: dans -> sans

File: docs/fr/api/render.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+- **Options:**
+
+Voir [les options](./options.md)
+
+- **Usage:**
+
+Rend un objet en chaîne de caractères et retourne un [cheerio wrapper](https://github.com/cheeriojs/cheerio).
+
+Cheerio est une bibliothèque de type jQuery pour parcourir le DOM dans Node.js. Elle possède une API similaire à celles de [`Wrapper`](wrapper/) Vue Test Utils .
```
Comment (cexbrayat) [inline docs/fr/api/render.md]: celles -> celle

File: docs/fr/api/wrapper-array/README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Propriétés
+
+### `wrappers`
+
+`array` (lecture seulement): les `Wrappers` contenus dans le `WrapperArray`
+
+### `length`
+
+`number` (lecture seulement): le nombre de `Wrappers`constenus dans le `WrapperArry`
```
Comment (cexbrayat) [inline docs/fr/api/wrapper-array/README.md]: constenus -> contenus (et espace manquant)

File: docs/fr/api/wrapper-array/at.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+## at
+
+Retourne `Wrapper` à `index` passé. Utilise une numérotation basée sur les zéros (c'est-à-dire que le premier élément est à l'index 0).
```
Comment (cexbrayat) [inline docs/fr/api/wrapper-array/at.md]: Retourne *le* Wrapper?

File: docs/fr/api/wrapper-array/exists.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+## exists
+
+Vérifie que `WrapperArray` existe.
+
+Renvoie false si elle est appelée sur un `WrapperArray` sans objets Wrapper`, ou si l'un d'entre eux n'existe pas.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper-array/exists.md]: manque une backtick devant Wrapper

File: docs/fr/api/wrapper/README.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
+# Wrapper
+
+Vue Test Utils est une API basée sur un wrapper
+
+Un `wrapper`est un objet qui contient un composant ou vnode monté et des méthodes pour tester le composant or vnode
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/README.md]: manque un espace

Comment (cexbrayat) [reply docs/fr/api/wrapper/README.md]: or -> ou

File: docs/fr/api/wrapper/README.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+<div class="vueschool"><a href="https://vueschool.io/lessons/the-wrapper-object?friend=vuejs" target="_blank" rel="sponsored noopener" title="Learn about the Wrapper object in a FREE video lesson from Vue School">Découvrez l'objet Wrapper dans une leçon vidéo GRATUITE de Vue School</a></div>
+
+## Propriétés
+
+### `vm`
+
+`Component`(lecture seule): c'est l'instance `Vue`. Vous pouvez accéder à toutes les [méthodes et propriétés de l'instance d'un vm](https://vuejs.org/v2/api/#Instance-Properties) avec un `wrapper.vm`. Ceci n'existe que pour le wrapper du composant Vue ou le wrapper du composant Vue liant les éléments HTMLE.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/README.md:13]: HTMLE -> HTML

File: docs/fr/api/wrapper/get.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
+## get
+
+::: warning Avertissement de déprédation
+L'utilisation de `get`pour rechercher un Composant est dépréciée et sera supprimée. Utilisé plutôt [`getCompoenent`](./getComponent.md)
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/get.md]: manque un espace
Utilisé -> Utilisez
getCompoenent -> getComponent

File: docs/fr/api/wrapper/isVisible.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+## isVisible
+
+Affirmer que `Wrapper` est visible.
+
+Retourne `false` si un vieil élément a le style `display: none` ou `visibility: hidden`, est situé à l'intérieur de la balise <details> fermée ou possède un attribut caché.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/isVisible.md]: vieil?

File: docs/fr/api/wrapper/setChecked.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+import Foo from './Foo.vue'
+
+const wrapper = mount(Foo)
+const radioInput = wrapper.find('input[type="radio"]')
+radioInput.setChecked()
+```
+
+- **Note:**
+
+Lorsque vous essayez de mettre la valeur à state via `v-model` par `radioInput.element.checked = true ; radioInput.trigger('input')`, `v-model` n'est pas déclenché. Le `v-model` est déclenché par l'événement `change`.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/setChecked.md]: valeur à state?

File: docs/fr/api/wrapper/setProps.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+
+- **Arguments:**
+
+  - `{Object} props`
+
+- **Usage:**
+
+Définir les props de `Wrapper` `vm` et forcer à mettre à jour
+
+Définit les props `Wrapper` `vm` et force la mise à jour.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/setProps.md]: Répétition

File: docs/fr/api/wrapper/setSelected.md [hunk 17]
```diff
@@ -0,0 +1,26 @@
...
+
+const wrapper = mount(Foo)
+const options = wrapper.find('select').findAll('option')
+
+options.at(1).setSelected()
+```
+
+- **Note:**
+
+Lorsque vous essayez de mettre la valeur à state via `v-model` par `option.element.selected = true ; parentSelect.trigger('input')`, `v-model` n'est pas déclenché. Le `v-model` est déclenché par l'événement `change`.
```
Comment (cexbrayat) [inline docs/fr/api/wrapper/setSelected.md]: valeur à state ?

File: docs/fr/guides/common-tips.md [hunk 18]
```diff
@@ -0,0 +1,343 @@
...
+
+Lorsque vous utilisez les méthodes `mount` ou `shallowMount`, vous pouvez vous attendre à ce que votre composant réponde à tous les événements du cycle de vie. Cependant, il est important de noter que les méthodes `beforeDestroy` et `destroyed` _ne seront pas déclanchées_ à moins que le composant ne soit détruit manuellement en utilisant `Wrapper.destroy()`.
+
+De plus, le composant ne sera pas automatiquement détruit à la fin de chaque spécification, et c'est à l'utilisateur d'arrêter ou de nettoyer manuellement les tâches qui continueront à s'exécuter (`setInterval` ou `setTimeout`, par exemple) avant la fin de chaque spec.
+
+### Écrire des tests asynchrones (nouveau)
+
+Par défaut, les mises à jour des lots de Vue s'effectuent de manière asynchrone (à la prochaine "cloche"). Ceci afin d'éviter les re-rendus DOM inutiles, et les calculs des watchers ([voir les docs](https://vuejs.org/v2/guide/reactivity.html#Async-Update-Queue) pour plus de détails).
```
Comment (cexbrayat) [inline docs/fr/guides/common-tips.md]: des lots de Vue?
"cloche" me semble bizarre aussi.

File: docs/fr/guides/common-tips.md [hunk 19]
```diff
@@ -0,0 +1,343 @@
...
+})
+```
+
+Vous pouvez également mettre à jour les props d'un composant déjà monté avec la méthode `wrapper.setProps({})`.
+
+\_Pour une liste complète des options, vueillez consulter la [section des options de montage](../api/options.md) de la documentation.
+
+### Simulation de transition
+
+Bien qu'appeler `await Vue.nextTick()` fonctionne bien pour la plupart des cas d'utilisation, il y a certaines situations où des solutions de contournement supplémentaire sont nécessaires. Ces problèmes seront résolus avant que la bibliothèque `vue-test-utils`ne sorte de la version bêta. Un exemple est celui des composants de test unitaire avec le wrapper `<transition>` fourni par Vue.
```
Comment (cexbrayat) [inline docs/fr/guides/common-tips.md]: manque un espace

File: docs/fr/guides/common-tips.md [hunk 20]
```diff
@@ -0,0 +1,343 @@
...
+
+  await wrapper.setData({
+    show: false
+  })
+
+  expect(wrapper.text()).not.toMatch(/Foo/)
+})
+```
+
+Cela remplace le comportement par défaut du composant `<transition>` et rend les enfants dès que la condition booléenne pertinente change, par opposition à l'application de classes CSS, qui est la façon dont le composant `<transition>` de Vue fonctionne.
```
Comment (cexbrayat) [inline docs/fr/guides/common-tips.md]: rend -> affiche

File: docs/fr/guides/dom-events.md [hunk 21]
```diff
@@ -0,0 +1,216 @@
...
+test('triggers a click', async () => {
+  const wrapper = mount(MyComponent)
+
+  await wrapper.find('button').trigger('click')
+})
+```
+
+### Les Options
+
+La méthode `trigger` prend en option un objet optionnel `options`. Les propriétés dans cet objet `options` sont ajoutés à l'événement.
```
Comment (cexbrayat) [inline docs/fr/guides/dom-events.md]: ajoutés -> ajoutées

File: docs/fr/guides/testing-async-components.md [hunk 22]
```diff
@@ -0,0 +1,144 @@
...
+2. Comportement asynchrone en dehors de Vue
+
+### Mises à jour appliquées par Vue
+
+Visualiser les lots en attente de mises à jour du DOM et les appliquer de manière asynchrone pour éviter les rendus inutiles causés par de multiples mutations de données.
+
+_Vous pouvez en savoir plus sur les mises à jour asynchrones dans la [Vue docs](https://vuejs.org/v2/guide/reactivity.html#Async-Update-Queue)_
+
+En pratique, cela signifie qu'après la mutation d'une propriété réactive, pour affirmer que le changement a été apporté, votre test doit attendre pendant que Vue effectue les mises à jour.
+Un autre moyen est d'utiliser `await Vue.nextTick()`, mais un moyen plus facile et plus propre est de simplement `await`(attendre) la méthode avec laquelle vous avez muté l'état, comme `trigger`
```
Comment (cexbrayat) [inline docs/fr/guides/testing-async-components.md]: manque un espace

File: docs/fr/guides/testing-async-components.md [hunk 23]
```diff
@@ -0,0 +1,144 @@
...
+  expect(wrapper.text()).toBe('value')
+})
+```
+
+Cette même technique peut être appliquée aux actionx de Vuex, qui retournent une promesse par défaut.
+
+#### Pourquoi ne pas se contenter de `await button.trigger()` ?
+
+Comme expliqué ci-dessus, il y a une différence entre le temps nécessaire à Vue pour mettre à jour ses composants,
+et le temps qu'il faut pour qu'une promesse, comme celle d'`axios` soit tenue.
```
Comment (cexbrayat) [inline docs/fr/guides/testing-async-components.md]: tenue -> résolue

File: docs/fr/guides/useful-libraries-for-testing.md [hunk 24]
```diff
@@ -0,0 +1,22 @@
+## Bibliothèques utiles pour les tests
+
+Vue Test Utils fournit des méthodes pour tester les composants de Vue. Les membres de la communauté ont également écrit quelques bibliothèques supplémentaires qui soit étendent les `vue-test-utils` avec des méthodes supplémentaires utiles, soit fournissent des outils pour tester d'autres choses trouvées dans les applications Vue.
+
+### Bibliothèque de test de Vue
+
+[Vue Testing Library](https://github.com/testing-library/vue-testing-library) est un ensemble d'outils visant à tester les composants sans se fier aux détails de la mise en œuvre. Conçu dans un souci d'accessibilité, son approche permet également de remanier facilement les composants.
+
+It is built on top of Vue Test Utils.
```
Comment (cexbrayat) [inline docs/fr/guides/useful-libraries-for-testing.md]: a supprimer

File: docs/fr/guides/using-with-typescript.md [hunk 25]
```diff
@@ -0,0 +1,157 @@
+## Utilisation avec TypeScript
+
+> Un exemple de projet pour cette installation est disponible sur [GitHub](https://github.com/vuejs/vue-test-utils-typescript-example).
+
+TypeScript est un sur-ensemble populaire de JavaScript qui ajoute des types et des classes en plus des JS ordinaires. Vue Test Utils inclut les types dans le paquet distribué, donc il fonctionne bien avec TypeScript.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md]: des *fonctionnalités* JS ?

File: docs/fr/guides/using-with-typescript.md [hunk 26]
```diff
@@ -0,0 +1,157 @@
...
+$ npm install -g @vue/cli
+```
+
+Et créer un projet avec :
+
+```shell
+$ vue create hello-world
+```
+
+Dans l’invite du CLI, choisissez de `Manually select features`, sélectionnez TypeScript et appuyez sur la touche enter. Cela créera un projet avec TypeScript déjà configuré.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md]: de -> l'option

File: docs/fr/guides/using-with-typescript.md [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+
+::: tip NOTE
+Si vous souhaitez obtenir un guide plus détaillé sur la configuration de Vue avec TypyScript, consultez le [guide de démarrage de Vue avec TypeScript](https://github.com/Microsoft/TypeScript-Vue-Starter).
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md]: TypyScript -> TypeScript

File: docs/fr/guides/using-with-typescript.md [hunk 28]
```diff
@@ -0,0 +1,157 @@
...
+:::
+
+L'étape suivante consiste d'ajouter Jest au projet.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md]: d'ajouter -> à ajouter

File: docs/fr/guides/using-with-typescript.md [hunk 29]
```diff
@@ -0,0 +1,157 @@
...
+
+### Mise en place de Jest
+
+Jest est un outil de test développer par Facebook, visant à fournir une solution de test unitaire sur batterie. Vous pouvez en savoir plus sur Jest en consultant sa [documentation officielle](https://jestjs.io/).
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md:33]: développer -> développé
sur batterie -> avec tout le nécessaire?

File: docs/fr/guides/using-with-typescript.md [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+    const msg = 'new message'
+    const wrapper = shallowMount(HelloWorld, {
+      propsData: { msg }
+    })
+    expect(wrapper.text()).toMatch(msg)
+  })
+})
+```
+
+C'est tout ce qui faut faire pour que TypeScript et Vue Test Utils travaillent ensemble !
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-typescript.md:152]: qui-> qu'il

File: docs/fr/guides/using-with-vue-router.md [hunk 31]
```diff
@@ -0,0 +1,83 @@
...
+const wrapper = shallowMount(Component, {
+  mocks: {
+    $route
+  }
+})
+
+wrapper.vm.$route.path // /some/path
+```
+
+### Les gotchas commun
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-vue-router.md:77]: gotchas communs -> problèmes courants

File: docs/fr/guides/using-with-vue-router.md [hunk 32]
```diff
@@ -0,0 +1,83 @@
...
+
+L'installation de Vue Router ajoute `$route` et `$router` comme propriétés en lecture seule sur le prototype de Vue.
+
+Cela signifie que tous les futurs tests qui tenteront de simuler `$route` et `$router` echoueront
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-vue-router.md:81]: echoueront -> échoueront

File: docs/fr/guides/using-with-vuex.md [hunk 33]
```diff
@@ -0,0 +1,392 @@
+# Utilisation avec Vuex
+
+Dans ce guide, nous verrons comme tester Vuex dans les composants avec Vue Test Utils, et comment aborder le test d'un sore de Vuex.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-vuex.md:3]: comme -> comment

File: docs/fr/guides/using-with-vuex.md [hunk 34]
```diff
@@ -0,0 +1,392 @@
...
+})
+```
+
+Que se passe-t-il ici ? Tout d'abord, nous disons à Vue d'utiliser Vuex avec la méthode `localVue.use`. C'est juste un emballage autour de `Vue.use`.
+
+Nous faisons ensuite un magasin fictif en appelant `new Vuex.Store` avec nos valeurs fictives. Nous lui transmettons seulement les actions, puisque c'est tout ce qui nous intéresse.
+
+Les actions sont des [fonctions fictives de jest](https://jestjs.io/docs/en/mock-functions.html). Ces fonctions fantaisie nous donnent des méthodes pour affirmer si les actions ont été appelées ou non.
+
+Nous pouvons alors affirmer dans nos tests que le talon d'action a été appelé au moment prévu.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-vuex.md:101]: talon -> fonction fictive?

File: docs/fr/guides/using-with-vuex.md [hunk 35]
```diff
@@ -0,0 +1,392 @@
...
+  const localVue = createLocalVue()
+  localVue.use(Vuex)
+  const store = new Vuex.Store(cloneDeep(storeConfig))
+  expect(store.getters.evenOrOdd).toBe('even')
+  store.commit('increment')
+  expect(store.getters.evenOrOdd).toBe('odd')
+})
+```
+
+Notez que nous utilisons `cloneDeep` pour cloner la configuration du store avant de créer un store avec lui. C'est parce que Vuex fait muter l'option d'options utilisé pour créer le store. Pour s'assurer que nous avons un store propre dans chaque test nous devons cloner l'objet `storeConfig`.
```
Comment (cexbrayat) [inline docs/fr/guides/using-with-vuex.md:377]: répétition d'options

File: docs/fr/installation/semantic-versioning.md [hunk 36]
```diff
@@ -0,0 +1,3 @@
+## Version sémantique
+
+Vue Test Utils suit [Semantic Versioning](https://semver.org/) dans tous ses projets officiels pour les fonctionnalités et le comportement documentés. Pour les comportementsnon documentés ou les internes exposés, les changements sont décrits dans les [notes de version](https://github.com/vuejs/vue-test-utils/releases).
```
Comment (cexbrayat) [inline docs/fr/installation/semantic-versioning.md:3]: manque un espace
ou les internes  exposés?

File: docs/fr/installation/using-other-test-runners.md [hunk 37]
```diff
@@ -0,0 +1,60 @@
+## Utilisation d'autres testeurs
+
+### Fonctionnement de Vue Test Utils avec Karma
+
+[Karma](http://karma-runner.github.io/) est un programme de test qui lance des navigateurs, effectue des tests et nous les rapporte.
+
+En plus de Karma, vous pouvez utiliser le cadre [Mocha](https://mochajs.org/) pour écrire les tests, et la bibliothèque [Chai](http://chaijs.com/) pour les assertions des tests. Vous pouvez également consulter [Sinon](http://sinonjs.org/) pour créer des spies et des stubs
```
Comment (cexbrayat) [inline docs/fr/installation/using-other-test-runners.md:7]: cadre -> la bibliothèque

File: docs/fr/installation/using-other-test-runners.md [hunk 38]
```diff
@@ -0,0 +1,60 @@
...
+    preprocessors: {
+      '**/*.spec.js': ['webpack', 'sourcemap']
+    }
+  })
+}
+```
+
+### Fonctionnement de Vue Test Utils avec mocha-webpack
+
+Une autre stratégie pour tests les SFC consiste à compiler tous nos tests via la webpack et à les exécuter ensuite dans un testeur. L'avantage de cette approche est qu'elle nous donne un support complet pour toutes les fonctionnalités du webpack et du `vue-loader`, donc nous n'avons pas à faire de compromis dans notre code source.
```
Comment (cexbrayat) [inline docs/fr/installation/using-other-test-runners.md]: la webpack -> webpack
de webpack

File: docs/fr/installation/using-other-test-runners.md [hunk 39]
```diff
@@ -0,0 +1,60 @@
...
+
+### Fonctionnement de Vue Test Utils sans étape de construction
+
+Alors qu'il est courant de construire des applications Vue en utilisant des outils tels que [webpack](https://webpack.js.org/) pour regrouper l'application, `vue-loader` pour exploiter les composants de fichiers uniques, il est possible d'uiliser beaucoup moins les Vue Test Utils . Les exigences minimales pour les Vue Test Utils, en dehors de la bibliothèque elle-même, sont les suivantes :
+
+- Vue
+- [vue-template-compiler](https://github.com/vuejs/vue/tree/dev/packages/vue-template-compiler#readme)
+- a DOM (be it [jsdom](https://github.com/jsdom/jsdom) in a Node environment, or the DOM in a real browser)
+
+Notez que `jsdom`(ou toute autre implémentation de DOM) doit être requis avant les Vue Test Utils, car il s'attend à ce qu'un DOM (vrai DOM, ou JSDOM) existe.s
```
Comment (cexbrayat) [inline docs/fr/installation/using-other-test-runners.md]: manque un espace

Comment (cexbrayat) [reply docs/fr/installation/using-other-test-runners.md]: existe.s -> existe.

File: docs/fr/installation/using-with-jest.md [hunk 40]
```diff
@@ -0,0 +1,119 @@
+## Utilisation des outils de Vue Test Utils avec Jest (recommandé)
+
+Jest est un outils de test développé par Facebook, visant à fournir une solution de test unitaire sur batterie. Vous pouvez en savoir plus sur Jest en consultant sa [documentation officielle](https://jestjs.io/).
```
Comment (cexbrayat) [inline docs/fr/installation/using-with-jest.md]: sur batteries -> avec tout le nécessaire fourni?

File: docs/fr/installation/using-with-jest.md [hunk 41]
```diff
@@ -0,0 +1,119 @@
...
+
+### Installation Manuelle
+
+Après avoir configuré Jest, la première chose à faire est d'installer Vue Test Utils et [`vue-jest`](https://github.com/vuejs/vue-jest) pour traiter les mono-fichiers.
+
+```bash
+$ npm install --save-dev @vue/test-utils vue-jest
+```
+
+Ensuite, vous devez dire à Jest de transformer les fichiers `.vue` en utilisant `.vu-jest`. Vous pouvez le faire en ajoutant la configuration suivante dans le `package.json` ou dans un [fichier de configuration de Jest](https://jestjs.io/docs/en/configuration) :
```
Comment (cexbrayat) [inline docs/fr/installation/using-with-jest.md]: .vu-jest -> vue-jest

File: docs/fr/installation/using-with-jest.md [hunk 42]
```diff
@@ -0,0 +1,119 @@
...
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/src/$1"
+    }
+  }
+}
+```
+
+### Couverture du code
+
+Jest peut être utilisé pour générer des rapports de couverture dans plusieur formats. Voici un exemple simple pour commencer :
```
Comment (cexbrayat) [inline docs/fr/installation/using-with-jest.md]: plusieurs

**Review by cexbrayat — COMMENTED** [review]
@phrobinet Thanks for your contribution!
I spotted a few typos.

Note: I'm not a huge fan of "affirmation"  and "affirmer" (for "assertion" and "assert" in English I guess). I would use "vérification" and "vérifier".

File: docs/fr/guides/using-with-typescript.md (hunk 25 above)
Comment (phrobinet) [reply docs/fr/guides/using-with-typescript.md]: J'ai un peu remodelé ma version: 

TypeScript est un superset populaire de JavaScript qui ajoute des types et des classes au dessus du JS normal. Vue Test Utils inclut les types dans le package distribué, il fonctionne donc bien avec TypeScript.

**Comment by phrobinet** [comment]
> 
> 
> @phrobinet Thanks for your contribution!
> I spotted a few typos.
> 
> Note: I'm not a huge fan of "affirmation" and "affirmer" (for "assertion" and "assert" in English I guess). I would use "vérification" and "vérifier".

I'm not a fan of this word either but I couldn't find the right one and "verification" is much better, thanks

**Comment by lmiller1990** [comment]
We can merge once we figure out why Netlify is failing 🤔

**Comment by lmiller1990** [comment]
I see. We need to configure Vuepress to support a new translation. In this file: https://github.com/vuejs/vue-test-utils/blob/dev/docs/.vuepress/config.js

Are you able to do that @phrobinet, or if you don't have time, we can look for someone else to do it (or I can look into it, quite busy fixing 100+ other issues, though.

**Comment by phrobinet** [comment]
[configure.md](https://github.com/vuejs/vue-test-utils/files/6740108/configure.md)
This is a configuration of the configure file so that it supports a new translation. Tell me if I need to make any changes to this file?

**Comment by lmiller1990** [comment]
I have no idea how it works, we will just need to play around with it. I'll give it a try now. `yarn vuepress build` fails - need to make it not fail.

**Comment by lmiller1990** [comment]
Fixed some problems but still getting

wait Rendering static HTML...
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[Vue warn]: Error in render: "TypeError: Cannot read property 'match' of undefined"

found in

---> <SidebarLinks>
       <Sidebar>
         <Layout>
           <Layout>
             <GlobalLayout>
               <Root>
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[vuepress] No matching page found for sidebar item "/fr/api/components/"
[Vue warn]: Error in render: "TypeError: Cannot read property 'match' of undefined"


I don't know why! 🤔

**Comment by phrobinet** [comment]
Sorry to answer you so late but I'm back from holiday.

But then I don't understand what is going on 🤨. 
This is the same configuration I use in my vue press projects and it's the same configuration as for the other languages.
I don't see what I've missed on this configuration. 
Maybe you know someone who could answer this question?

**Comment by phrobinet** [comment]
Hello,
I have understood where the problem with the sidebar was and I have just corrected it. And by the same occasion, I modified two pages (README) which are displayed badly in the French version
Normally it should work

**Comment by lmiller1990** [comment]
Nice job! Just need to run prettier so CI can build, and we can merge. Thanks!

**Comment by lmiller1990** [comment]
Merged!! Netlify should auto deploy.

**Pull request merged**